kubetest/e2e.go

```diff
@@ -565,7 +564,7 @@ func nodeTest(nodeArgs []string, testArgs, nodeTestArgs, project, zone string) e
 	return control.FinishRunning(exec.Command("go", runner...))
 }
 
-func kubemarkTest(testArgs []string, dump string, o options, deploy deployer) error {
+func kubemarkUp(dump string, o options, deploy deployer) error {
```

Comment:
+1 to @krzyzacy suggestion to move all kubemark-related function to a separate file.

Reply:
Though maybe a separate PR, to avoid mixing logic changes with mechanical ones.